src/java.base/share/classes/sun/security/tools/keytool/Resources_de.java

```diff
@@ -304,10 +304,12 @@ public class Resources_de extends java.util.ListResourceBundle {
                 "Das Angeben von -keysize zum Generieren von EC-Schl\u00FCsseln ist veraltet. Verwenden Sie stattdessen \"-groupname %s\"."},
         {"Key.pair.not.generated.alias.alias.already.exists",
                 "Schl\u00FCsselpaar wurde nicht generiert. Alias <{0}> ist bereits vorhanden"},
-        {"Generating.keysize.bit.keyAlgName.key.pair.and.self.signed.certificate.sigAlgName.with.a.validity.of.validality.days.for",
-                "Generieren von Schl\u00FCsselpaar (Typ {1}, {0} Bit) und selbst signiertem Zertifikat ({2}) mit einer G\u00FCltigkeit von {3} Tagen\n\tf\u00FCr: {4}"},
-        {"Generating.keysize.bit.keyAlgName.key.pair.and.a.certificate.sigAlgName.issued.by.signerAlias.with.a.validity.of.validality.days.for",
-                "Generieren von Schl\u00FCsselpaar (Typ {1}, {0} Bit) und Zertifikat ({2}), ausgestellt von <{3}>, mit einer G\u00FCltigkeit von {4} Tagen\n\tf\u00FCr: {5}"},
+        {"size.bit.alg",
+                "%1$d-Bit %2$s"},
+        {"Generating.full.keyAlgName.key.pair.and.self.signed.certificate.sigAlgName.with.a.validity.of.days.for",
+                "Schl\u00FCsselpaar {0} und selbstsigniertes Zertifikat ({1}) werden mit einer G\u00FCltigkeit von {2} Tagen generiert\n\tf\u00FCr: {3}"},
```

Comment:
Putting `{0}` (which is size and type) after "Schlüsselpaar" sounds strange. I understand you don't want to put `{0}` in parentheses because the value itself might contain parentheses (For example, `384-Bit EC (secp384r1)`). Maybe we can add a "mit" in between?

Also, Why change from active voice in the present continuous tense to passive voice in the simple present tense?